proposals/0099-8-bit-types.md

```diff
@@ -0,0 +1,67 @@
+<!-- {% raw %} -->
+
+# Refined `cbuffer` Contexts
+
+## Instructions
+
+* Proposal: [0099](0099-8-bit-types.md)
+* Author(s): [Matthäus Chajdas](https://github.com/anteru)
+* Sponsor: TBD
+* Status: TBD
+* Planned Version: 202x
```

Comment:
I think that this is likely a shader model proposal rather than purely a language one?